validator/src/main.rs

```diff
@@ -2114,6 +2114,12 @@ pub fn main() {
         .collect::<HashSet<_>>()
         .into_iter()
         .collect::<Vec<_>>();
+    for addr in &entrypoint_addrs {
+        if !socket_addr_space.check(addr) {
+            eprintln!("invalid entrypoint address: {}", addr);
```

Comment:
I'd like to give the operator more informative feedback here, but it looks like that's not possible (in a forward compatible way) without some surgery on `SocketAddrSpace::check` to return a failure reason :thinking:

Reply:
`--allow-private-addr` is hidden flag. so I guess the assumption is that the error has a very specific and limited audience.
But feel free to suggest any edits if adds more context.